docs/error-reporting/advanced/morgue.md

```diff
@@ -0,0 +1,673 @@
+
+### Setup
+
+Configure the initial organization and user for an on-premise version of coronerd.
+
+```bash
+morgue setup <coronerd_url>
+```
+
+If you are using an on-premise version of coronerd, use the `morgue setup` command to configure the initial organization and user. Replace `<coronerd_url>` with the URL of your coronerd server. For example, if the server is `backtrace.mycompany.com`, you would run:
+
+```bash
+morgue setup http://backtrace.mycompany.com
+```
+
+We recommend resetting your password after enabling SSL (done by configuring your certificates).
+
+### Nuke
+
+Delete an object and all its dependencies.
+
+```bash
+morgue nuke --universe=<universe>
+```
+
+Use this command to nuke an object and all its dependencies. Make sure to back up your data before using this command.
+
+### Token
+
+Manage API tokens for authentication and authorization.
+
+```bash
+morgue token  [create | list | delete] [--project=<project>] [--universe=<universe>]
+```
+
+The `morgue token` command without any additional parameters lists the available commands for managing API tokens.
+
+#### Create Token
+
+Create a new API token.
+
+```bash
+morgue token create --project=<project>
+    [--capability=<capability> ...]
+```
+
+The `--capability` option specifies the capabilities of the API token. Multiple capabilities can be specified by using the `--capability` option multiple times or by using a comma-separated list. Valid capabilities include:
+
+- `symbol:post`: Enable symbol uploads with the specified API token.
+- `error:post`: Enable error and dump submission with the specified API token.
+- `query:post`: Enable queries to be issued using the specified token.
+
+#### List Token
+
+Lists the API tokens in the specified universe or project.
+
+```bash
+morgue token list [--universe=<universe>] [--project=<project>]
+```
+
+##### Example
+
+Allows you to perform queries on object metadata. You can perform either selection queries or aggregation queries, but not both at the same time.
+
+```bash
+$ morgue token list --universe=my_universe --project=my_project
+```
+
+You may pass `--verbose` in order to get more detailed query performance data.
+
+#### Delete Token
+
+Deletes the specified token by substring or exact match.
+
+```bash
+morgue token delete
+```
+
+### User
+
+Modify users.
+
+```bash
+morgue user reset [--universe=<universe>] [--user=<user>] [--password=<password>]
+```
+
+Currently, it can only be used to reset user passwords. If the `--user` or `--password` options are not specified, the command will prompt for the user and password.
```

Comment:
```suggestion
It can only be used to reset user passwords. If the `--user` or `--password` options are not specified, the command will prompt for the user and password.
```